eng/common/scripts/Verify-Links.ps1

```diff
@@ -167,6 +171,11 @@ function CheckLink ([System.Uri]$linkUri)
   }
   $checkedLinks[$linkUri] = $true;
 }
+$GithubRegex = "($sourceRepoUrl/blob/)([a-zA-Z0-9/-/_]+)(/.*)"
+$ReplacementPattern = "`${1}$sourceCommit`$3"
+function ReplaceGithubLink([string]$originLink) {
```

Comment:
We should only do this if the $sourceCommit isn't empty.

Reply:
Is there a chance $sourceCommit return empty?